frontend/src/constants/errors.ts

```diff
@@ -5,7 +5,45 @@
 
 export const NO_RESULT = "NO_RESULT"
 export const SERVER_TIMEOUT = "SERVER_TIMEOUT"
+export const ECONNABORTED = "ECONNABORTED"
 
-export const errorCodes = [NO_RESULT, SERVER_TIMEOUT] as const
+export const ERR_UNKNOWN = "ERR_UNKNOWN"
+
+export const customErrorCodes = [
+  NO_RESULT,
+  SERVER_TIMEOUT,
+  ECONNABORTED,
+  ERR_UNKNOWN,
+] as const
+
+/**
+ * The error codes Axios uses.
+ * @see https://github.com/axios/axios/blob/9588fcdec8aca45c3ba2f7968988a5d03f23168c/lib/core/AxiosError.js#L57C2-L71
+ */
```

Comment:
Is there any chance this list could change? Is there any way to reference it directly in Axios? 😮

Reply:
This could change, but there is no way of importing the list of error kinds directly from Axios :(